.travis.yml

```diff
@@ -44,9 +44,9 @@ matrix:
     - cd kitchen-tests
     script:
 # FIXME: we should fix centos-6 against AWS and then enable it here
-    - if [ "$TRAVIS_SECURE_ENV_VARS" = "true" ]; then bundle exec kitchen test ubuntu; fi
+    - if [ "$TRAVIS_SECURE_ENV_VARS" = "true" ]; then bundle exec kitchen test ubuntu installthings --concurrency 8; fi
```

Comment:
this just prevents the webapp stuff from running on centos as those tests are broken